app/code/local/Magmodules/Webwinkelconnect/Model/Observer.php

```diff
@@ -185,4 +185,49 @@ public function sendSyncUrl(): void {
             $curl->close();
         }
     }
+
+
+    public function addOrderDataJsonThankYouPage(): void {
+        if (Mage::app()->getFrontController()->getAction()->getFullActionName() != 'checkout_onepage_success') {
```

Comment:
I wonder if it is possible to extend the `checkout_onepage_success`? instead of with this observer